ShoppingM8-be/src/main/java/edu/pw/shoppingm8/list/ListService.java

```diff
@@ -0,0 +1,13 @@
+package edu.pw.shoppingm8.list;
+
+import edu.pw.shoppingm8.list.api.dto.ListDto;
+
+public interface ListService {
```

Comment:
The service is lacking a method that returns a collection of user's lists. Please, create one.

Reply:
Does it make sense to add it? I see no case where we would need to get a collection of lists owned by the user, only a collection of lists where user is participating, and that is not a part of this task.

Reply:
alright